flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/SequenceFileSerializerFactory.java

```diff
@@ -28,14 +28,15 @@
   private static final Logger LOGGER =
       LoggerFactory.getLogger(SequenceFileSerializerFactory.class);
 
-  private SequenceFileSerializerFactory(){}
-
   /**
    * {@link Context} prefix
    */
   static final String CTX_PREFIX = "writeFormat.";
 
-  @SuppressWarnings("squid:S1166")
+  private SequenceFileSerializerFactory()
+  {}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)